addons/mail/static/src/core/common/message_in_reply.xml

```diff
@@ -9,9 +9,9 @@
             }">
                 <span class="d-inline-flex align-items-center text-muted opacity-75" t-att-class="{ 'cursor-pointer opacity-100-hover': props.onClick }" t-on-click="() => this.props.onClick?.()">
                     <t t-if="!props.message.parentMessage.isEmpty">
-                        <img class="o-mail-MessageInReply-avatar me-2 o-rounded-bubble o_object_fit_cover" t-att-src="authorAvatarUrl" t-att-title="props.message.parentMessage.author?.name ?? props.message.parentMessage.email_from" alt="Avatar"/>
+                        <img class="o-mail-MessageInReply-avatar me-2 o-rounded-bubble o_object_fit_cover" t-att-src="authorAvatarUrl" t-att-title="props.message.parentMessage.authorName ?? props.message.parentMessage.email_from" alt="Avatar"/>
```

Comment:
Same here with `title`: remove the `email_from` branch (already handled in `authorName`.